jbod_test.py

```
@@ -0,0 +1,125 @@
+from dtest import Tester, debug, DISABLE_VNODES
+from ccmlib.node import Node, NodeError, TimeoutError
+from cassandra import ConsistencyLevel, Unavailable, ReadTimeout
+from cassandra.query import SimpleStatement
+from tools import since
+import subprocess
+import os
+import time
+import shutil
+
+@since('3.0')
+class TestJBOD(Tester):
+
+    def __init__(self, *args, **kwargs):
+        self.ignore_log_patterns = [
+            # This one occurs when trying to send the migration to a
+            # node that hasn't started yet, and when it does, it gets
+            # replayed and everything is fine.
+            r"doesn't exist",
+        ]
+        Tester.__init__(self, *args, **kwargs)
+        self.allow_log_errors = True
```

This should probably be replaced with more patterns in `ignore_log_patterns`; ignoring all errors could let some failures slip past us.